script/sync-crls

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 
 # script/sync-crls: update the DOD CRLs and place them where authnid expects them
+set -e
+cd "$(dirname "$0")/.."
 
-rm -rf crl/ || true
+mkdir -p crl-tmp
```

Comment:
`crl-tmp` shouldn't be sticking around, but can we add it to the `.gitignore` just in case?